frontend/components/draft/lesson/code-block-viewer.tsx

```
@@ -0,0 +1,307 @@
+      };
+    }
+
+    const output =
+      result.run.stdout ||
+      result.run.output ||
+      "Code executed successfully (no output)";
+    return {
+      success: true,
+      output: output.trim(),
+    };
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)